src/schema/flags.rs

```diff
@@ -30,6 +30,18 @@ pub const INDEXED: SchemaFlagList<IndexedFlag, ()> = SchemaFlagList {
     tail: (),
 };
 
+#[derive(Clone)]
+pub struct NormedFlag;
+/// Flag to mark the field as indexed.
+///
+/// The `INDEXED` flag can only be used when building `IntOptions` (`u64`, `i64` and `f64` fields)
+/// Of course, text fields can also be indexed... But this is expressed by using either the
+/// `STRING` (untokenized) or `TEXT` (tokenized with the english tokenizer) flags.
+pub const NORMED: SchemaFlagList<NormedFlag, ()> = SchemaFlagList {
```

Comment:
I think we do not need a flag for normed. It is a rather advanced attribute.

On the other hand, we probably want INDEXED to imply NORMED for backward compatibility

Reply:
In other words, people who want to disable NORMED won't be able to use this syntactic sugar and will have to configure the field manually which is fine.